<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GastosApp</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>140</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>601</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>COSTE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDateEdit" name="dateEdit">
       <property name="locale">
        <locale language="Spanish" country="Spain"/>
       </property>
       <property name="displayFormat">
        <string>d/M/yyyy</string>
       </property>
       <property name="timeSpec">
        <enum>Qt::UTC</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Hipoteca</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Comunidad</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Garaje</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Luz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gas</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Agua</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Alimentación</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Telecomunicaciones</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hogar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Combustible</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Peajes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Impuestos</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Seguros</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TIPO</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>FECHA</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>DESCRIPCIÓN</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="saveBtn_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 175, 62);
font: 700 10pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>EDITAR GASTO</string>
    </property>
   </widget>
  </widget>
  <action name="actionSALIR">
   <property name="text">
    <string>SALIR</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
